### PR TITLE
Added api_readme.txt to project

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner @leogrim @jaredpetker

### CODE REVIEW

**Comment by andrewconner** [comment]
`kifi-backend/conf` is actually a directory with meaning for Play. Perhaps if you want this create a new one, like `/docs`?

File: kifi-backend/conf/api_readme.txt [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    id: PublicId[Organization]
+    ownerId: ExternalId[User]
+    handle: String
+    name: String
+    description: Option[String]
+    avatarPath: Option[String]
+    numMembers: Int
+    numLibraries: Int
+
+  LibraryCardInfo
```
Comment (andrewconner) [inline kifi-backend/conf/api_readme.txt:118]: This, which is basically Scala code, will become out of date without someone or something in charge of maintaining it (like a compiler).

**Comment by maxov** [comment]
I would also recommend using markdown with this, because it would look far nicer. I can convert it if you wish, but the syntax is not hard to learn

**Comment by andrewconner** [comment]
If you want documentation about endpoints, why not just have a test for each endpoint that demonstrates how to use it, and what to expect back? That way it'll be verified on every deploy.

**Pull request closed without merging**